### DISCUSSION TITLE
New builds feedback

### DISCUSSION THREAD

**Discussion by poogie** [celguar/mangosbot-bots#177] [General]
Hello.
Thanks all developers for the hard work.
I'm testing all the fresh builds in my SPP server installation (TBC). The server is up for months and no problems updating manually so far.
Replacing binaries / pdb, and updating the sql database via Heidi with the new sql files under sql/playerbot.
If you are interested in my feedback (from user perspective), just let me know.
Regards

**Comment by mostlikely4r** [comment]
Hey Poogie,

Always interested in hearing feedback. Either here or on our discord let us know your thoughts.

**Comment by poogie** [comment]
TBC:
Updated database for 12/13/2022 Release
world :
https://github.com/cmangos/tbc-db/releases/tag/latest   (tbc-world-db only)
realm:
https://github.com/cmangos/mangos-tbc/blob/master/sql/updates/realmd/s2455_01_realmd_platform.sql    (apply  sql update)

Used HeidiSQL for the process.

Everything is working fine

**Comment by poogie** [comment]
Since Tbc beta builds are now released for everyone via update, this thread is useless at this point.
I'll be reporting possible issues though the proper section. Anyway, I'll keep updating manually from this repo for testing.
Thanks for your patience and help

**Comment by celguar** [comment]
Beta builds are released only as beta builds. TBC Core update is released with latest bots code and it gonna stay like that for some time, unless some critical issues are found that need fixing

**Comment by celguar** [comment]
beta builds are built daily as usual
